src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index;
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        index = random.nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
Use `name()` for getting String representation of enum constants instead of `toString()`.